### PR TITLE
added marketing task

### PR DESCRIPTION
**Pull request by Lyla-Fischer**

### CODE REVIEW

**Comment by Lyla-Fischer** [comment]
@rishig review?

File: tasks/2017/spread-the-word.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of
+projects like Zulip. 
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites. Mention that you're doing this task on some stream, and a mentor will send you the instructions. You're free to decide to not do the task after reading the instructions. We expect completing all the tasks will take about 90 minutes.
```
Comment (lonerz) [inline tasks/2017/spread-the-word.md]: Maybe "You're free to decide to do or not to do the task after reading the instructions."

**Comment by Lyla-Fischer** [comment]
@zulipbot label "test"

**Comment by lonerz** [comment]
Testing final review label. @zulipbot label "ready for final review"

File: tasks/2017/spread-the-word.md (hunk 1 above)
Comment (rishig) [reply tasks/2017/spread-the-word.md]: A few minor notes:
* We line wrap to 80 chars to make it easy to see diffs and the output of `git grep`. In emacs I think you can do this with `Meta-q`; it's probably worth finding out what this is for your editor.
* some stream -> should be more specific, like "on foo stream, with topic bar"

**Comment by rishig** [comment]
Thanks @Lyla-Fischer! Left a minor comment above.

**Comment by Lyla-Fischer** [comment]
@zulipbot remove "ready for final review"

File: tasks/2017/spread-the-word.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+2. Think of 2-4 products you use, and upvote (“Like”) them. To get to a product,
+search for it in the “I want to replace” bar in the upper right. Zulip's profile
+is [here](https://alternativeto.net/software/zulip-chat-server/) 
+
+3. For each of the products in 2, consider their competitors, and go to those
+competitors pages and list those products as good alternatives. Zulip is
+considered by many to be a good alternative to
+[Slack](https://alternativeto.net/software/slack/)
+
+3. Send us a link to your profile (e.g.
```
Comment (derAnfaenger) [inline tasks/2017/spread-the-word.md]: For this and the preceding analogous lines: I think `us` should be more specific, e.g. put all the links and review pointers  in some text file and submit that file on the GCI app.

**Comment by Lyla-Fischer** [comment]
I messed up git. Working on fixing it.

**Comment by Lyla-Fischer** [comment]
Okay - so I was trying to rebase using command line instead of github's snazzy interface, and now it is listing all of the commits from master as part of the pull request. I'm going to close this pull request and see if opening a new one fixes it.

**Comment by Lyla-Fischer** [comment]
https://github.com/zulip/zulip-gci/pull/639

**Pull request closed without merging**